assets/src/main/java/bisq/asset/coins/BitcoinRebooted.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * This file is part of Bisq.
+ *
+ * Bisq is free software: you can redistribute it and/or modify it
+ * under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or (at
+ * your option) any later version.
+ *
+ * Bisq is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero General Public
+ * License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with Bisq. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package bisq.asset.coins;
+
+import bisq.asset.AddressValidationResult;
+import bisq.asset.AddressValidator;
+import bisq.asset.Coin;
+
+public class BitcoinRebooted extends Coin {
+
+    public BitcoinRebooted() {
+        super("Bitcoin Rebooted", "BOOT", new BitcoinRebootedAddressValidator());
```

Comment:
Please use Regex validator

Reply:
Do you have a preferred coin example I follow?  When I previously submitted I was told the code looked fine, there was just a minor conflict.  Want to make sure I submit something acceptable.

Reply:
https://github.com/bisq-network/bisq/blob/master/assets/src/main/java/bisq/asset/coins/Beam.java

Reply:
Modified per your request.  Also refreshed bisq.asset.Asset to avoid conflicts until your next refresh.